manifests/charts/istio-cni/values.yaml

```diff
@@ -44,6 +44,20 @@ cni:
   # Allow the istio-cni container to run in privileged mode, needed for some platforms (e.g. OpenShift)
   privileged: false
 
+  providers:
+    multus:
+      # Some k8s flavors (e.g. OpenShift) require that a NetworkAttachmentDefinition be present.
+      globalNetworkAttachmentDefinition:
+        # If enabled, a global NetworkAttachmentDefinition will be created in the namespace below. Note that the cluster
+        # needs to have this CRD (network-attachment-definitions.k8s.cni.cncf.io) installed beforehand.
+        enabled: false
+
+        # The namespace the NetworkAttachmentDefinition global resource will be created on. This resource will be referenced
+        # by all pods that have a sidecar. This is done by putting the annotation `k8s.v1.cni.cncf.io/networks: <ns>/istio-cni` into the pod.
+        # By default `default` namespace is allowed to be referenced by all namespaces.
+        # For more information on this, see https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/configuration.md#namespace-isolation
+        namespace: default
```

Comment:
If we don't expect users to use a different one - I would just remove it as an option.